resources/web/docs_js/components/console_widget.jsx

```diff
@@ -62,6 +62,56 @@ export class _ConsoleForm extends Component {
   }
 }
 
+export class _TryConsoleSelector extends Component {
+  componentWillMount() {
+    const defaultVals = omit(['langStrings', 'saveSettings', 'url_label', 'setting'], this.props);
+    this.setState(defaultVals);
+  }
+
+
+  render(props) {
+    const handleConfigureClick = (e) => {
+      e.preventDefault();
+      props.settingsModalAction();
+    }
+
+    return (
+      <div className="try_console_selector">
+        <h4>Try in Console</h4>
+        <p>We were unable to detect a running console server.</p>
+        <p>
+          <a
+            id="try_console_selector_try_cloud_button"
+            className="button btn-primary btn-small"
+            href="https://cloud.elastic.co/registration"
```

Comment:
```suggestion
            href="https://www.elastic.co/cloud/elasticsearch-service/signup?page=docs&placement=console-signup"
```

@KOTungseth do we still track signups with the `page=docs` query param? ([source](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L188))

Reply:
Nope. I don't the mechanism that tracks this data.

Reply:
This is (was?) something finance or marketing used. I can't remember.

Reply:
This was owned by CSG, but the specific people who tracked this are gone and did not pass off the information. I've asked others, but they are also unable to find the information.